mvlearn/embed/cca.py

```diff
@@ -109,6 +107,11 @@ def _fit(self, Xs):
             raise ValueError(
                 f"CCA accepts exactly 2 views but {self.n_views_}"
                 "were provided. Consider using MCCA for more than 2 views")
+        if not (type(self.n_components) == int and
```

Comment:
`type(..) == ` is evil. You should use isinstance and for testing if it's an int you should use numbers.Integral https://docs.python.org/fr/3/library/numbers.html#numbers.Integral

```
In [5]: type(np.int64(2)) == int
Out[5]: False

In [6]: import numbers

In [7]: isinstance(np.int64(2), numbers.Integral)
Out[7]: True
```